spec/spec_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+TEST_DIR = __dir__
+TMP_DIR  = File.expand_path("../tmp", TEST_DIR)
+
+require "jekyll"
+require File.expand_path("../lib/jekyll-commonmark.rb", TEST_DIR)
```

Comment:
can't we use `require_relative "jekyll-commonmark"` instead?

Reply:
#TIL 😻